lib/server.js

```diff
@@ -34,6 +34,7 @@ function handleResponse(opts, req, resp, response) {
         rh['access-control-allow-origin'] = '*';
         rh['access-control-allow-methods'] = 'GET';
```

Comment:
Semi-related thing: we support `HEAD` and `POST` on some endpoints too. May be we could allow them here too?